Resources/doc/reference/query_proxy.rst

```diff
@@ -27,3 +27,14 @@ The ``ProxyQuery`` object is used to add missing features from the original `Doc
     $proxyQuery->setMaxResults(10);
 
     $results = $proxyQuery->execute();
+
+* add query hint in advanced usage
```

Comment:
Doesn't belong in this list. The header of this list is:

> The `ProxyQuery` object is used to add missing features from the original `Doctrine Query` builder

But obviously this is not a missing feature.

Anyway, there is no need to document it at all IMO.